content/tor-browser-for-android/content.lr

```diff
@@ -0,0 +1,67 @@
+_model: topic
+---
+title: Tor Browser for Android
+---
+description: Learn about Tor Browser for Android and how to install it
+---
+seo_slug: tor-broser-for-android
+---
+body:
+
+##### INTRODUCTION - A LITTLE BIT OF HISTORY
+
+[Mobile browsing](https://www.theguardian.com/technology/2016/nov/02/mobile-web-browsing-desktop-smartphones-tablets) has increasingly become the primary way many people go online.
```

Comment:
We should avoid linking external resources to our documentation. 
First, we want our users to dive into our documentation.
Second, the external website may disappear, vanish or have a simple internal modification (like a CMS migration) breaking our documentation.
Third, this external resource is not localized, this means that non-English speaking users will not be able to read it.